.github/workflows/browser-tests.yml

```diff
@@ -77,8 +77,7 @@ jobs:
 
     - name: Build Admin
       if: env.ENVIRONMENT == 'browser-tests-local'
-      working-directory: ghost/admin
-      run: yarn build:dev
+      run: yarn workspace @tryghost/admin-x-settings build && yarn workspace ghost-admin build:dev
```

Comment:
```suggestion
      run: yarn nx run ghost-admin:build:dev
```

Reply:
This is automatically configured to build admin-x-settings if we run it through NX